app/middleware/helpers.js

```diff
@@ -0,0 +1,8 @@
+import Org from '../models/Org';
+
+async function getNewOrgId(name) {
```

Comment:
Add comment to describe what this function does and a `// TODO` for updating the id generation